waiter/static/js/runs.js

```diff
@@ -101,6 +109,8 @@ $(function() {
       // TODO: check if successful (what if user not logged in? redirect to login page?)
     }
   });
-
-  var myLineChart = new Chart($("#resource-chart")[0].getContext('2d'), createConfig());
+  // Get chart data.
+  $.getJSON("/api/runs/channels/" + channel_id + "/runs/", function(data) {
```

Comment:
@arvs Do you have a different configuration in your `config/urls.py` ?  

For me the endpoints are under `url(r'^api/', include('runs.urls'))`  not `url(r'^api/runs/', include('runs.urls'))` so I changed the js code to look for the data there.